ContextDesktop/src/main/java/com/github/migi_1/Context/screens/MainMenu.java

```diff
@@ -38,13 +38,13 @@
      * Creates the start screen.
      */
      private void createStartScreen() {
-         nifty.fromXml("com/github/migi_1/Context/resources/main_screen.xml", "start", mainMenuController);
+         nifty.fromXml("Screens/main_screen.xml", "start", mainMenuController);
```

Comment:
This should have been fixed before, so now is the right time:
Make a private static final attribute of this.